ts/store/reducers/instabug/instabugUnreadMessages.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Instabug message reducer
+ */
+import { getType } from "typesafe-actions";
+import { responseInstabugUnreadMessagesLoaded } from "../../actions/instabug";
+
+import { Action } from "../../actions/types";
+import { GlobalState } from "../types";
+
+export type InstabugUnreadMessagesState = Readonly<{
+  unreadMessages: number;
+}>;
+
+const INITIAL_STATE: InstabugUnreadMessagesState = {
+  unreadMessages: 0
+};
+
+const reducer = (
+  state: InstabugUnreadMessagesState = INITIAL_STATE,
+  action: Action
+): InstabugUnreadMessagesState => {
+  switch (action.type) {
+    case getType(responseInstabugUnreadMessagesLoaded):
+      return {
+        ...state,
```

Comment:
```suggestion
```
since we have only 1 field spreading state is not necessary

Reply:
Sorry, now it's ok